ui/src/components/Inventories/InventoryFormModal.js

```diff
@@ -0,0 +1,189 @@
+import Button from '@material-ui/core/Button'
+import Dialog from '@material-ui/core/Dialog'
+import DialogActions from '@material-ui/core/DialogActions'
+import DialogContent from '@material-ui/core/DialogContent'
+import DialogTitle from '@material-ui/core/DialogTitle'
+import Grid from '@material-ui/core/Grid'
+import React from 'react'
+import TextField from '../Form/TextField'
+import { Field, Form, Formik } from 'formik'
+import { MeasurementUnits } from '../../constants/units'
+import { Checkbox, FormControlLabel, MenuItem } from '@material-ui/core'
+import products from '../../ducks/products'
+
+const measurementUnitKeys = Object.keys(MeasurementUnits)
+const allProducts= products
```

Comment:
See previous comment in `InventoryLayout`: remove any products from here and above.